src/vs/server/node/remoteLanguagePacks.ts

```diff
@@ -44,3 +50,39 @@ export namespace InternalNLSConfiguration {
 		return candidate && typeof candidate._languagePackId === 'string';
 	}
 }
+
+export const getLocaleFromConfig = async (argvResource: string): Promise<string> => {
+	try {
+		const content = stripComments(await fs.promises.readFile(argvResource, 'utf8'));
+		return JSON.parse(content).locale;
+	} catch (error) {
+		if (error.code !== 'ENOENT') {
+			console.warn(error);
+		}
+		return 'en';
+	}
+};
+
+const stripComments = (content: string): string => {
```

Comment:
Looks like this is now exported and available here: https://github.com/microsoft/vscode/blob/f3a9245af988874dd1bae7281d904ff6133304c6/src/vs/base/common/json.ts#L1316